src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -323,6 +323,41 @@ public async Task<bool> IsCompatibleAsync(string subject, string schema)
             => await restService.TestLatestCompatibilityAsync(subject, schema).ConfigureAwait(continueOnCapturedContext: false);
 
 
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.IsCompatibleAsync(string, int, string)" />
+        /// </summary>
+        public async Task<bool> IsCompatibleAsync(string subject, int versionId, string schema)
+            => await restService.TestCompatibilityAsync(subject, versionId, schema).ConfigureAwait(continueOnCapturedContext: false);
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.GetCompatibilityAsync(string)" />
+        /// </summary>
+        public async Task<Compatibility> GetCompatibilityAsync(string subject)
+            => await restService.GetCompatibilityAsync(subject).ConfigureAwait(continueOnCapturedContext: false);
+
+
+        /// <summary>
+        ///     Refer to <see cref="Confluent.SchemaRegistry.ISchemaRegistryClient.SetCompatibilityAsync(string, Compatibility)" />
+        /// </summary>
+        public async Task<Compatibility> SetCompatibilityAsync(string subject, Compatibility compatibility)
```

Comment:
I think this should be `UpdateCompatibilityAsync` to match the java client and also because it more precisely describes what's going on.

I also think this shouldn't return `Compatibility` (even though this is what the java client does). I think it makes sense for the PUT api call to do this, and also for the `RestService` that backs this method to do it (since it's a direct mapping to the API), however this is a higher level abstraction, and I think returning compatibility does more to harm the API than good because people may think some other value may be returned.